compute/admin_guide/waas/deploy_waas/deployment_vpc_mirroring.adoc

```diff
@@ -157,6 +157,8 @@ Once the VPC configuration is saved, a CloudFormation template will be created a
 
 * *Deploying*: The WAAS rule is getting ready as the Observer is being deployed in the AWS instance and the session is being established between the Observer and the resources.
 
+You must use *Refresh* to view the updated status of the deployment.
+
```

Comment:
@divyabhushan - I suggest a different phrasing:
Using Refresh will update the deployment status.

Reply:
Thanks @e-shuster for your suggestion. The idea is to keep it in active voice. What do you think?

Reply:
I made a minor tweak to state it as *Refresh* to view the updated deployment status, 
and fixed the syntax.